src/cmd/root.go

```diff
@@ -60,37 +60,13 @@ func init() {
 	viper.BindEnv("api-timeout", "OPSLEVEL_API_TIMEOUT")
 	viper.BindEnv("workers", "OPSLEVEL_WORKERS", "OL_WORKERS")
 	cobra.OnInitialize(func() {
-		readConfig()
 		setupLogging()
 		setupOutput()
 		setupConcurrency()
 		setupAPIToken()
 	})
 }
 
-func readConfig() {
-	if cfgFile != "" {
-		if cfgFile == "." {
```

Comment:
Also, IIRC you can never parse `.` as an argument in the CLI, so this should have actually been `-`.